src/libpriv/rpmostree-postprocess.cxx

```diff
@@ -1890,16 +1889,31 @@ filter_xattrs_cb (OstreeRepo     *repo,
         }
     }
 
- out:
-  if (local_error)
-    {
+  return g_variant_ref_sink (g_variant_builder_end (&builder));
+}
+
+/* Filters out all xattrs that aren't accepted. */
+static GVariant *
+filter_xattrs_cb (OstreeRepo     *repo,
+                  const char     *relpath,
+                  GFileInfo      *file_info,
+                  gpointer        user_data)
+{
+  g_assert (relpath);
+
+  GVariantBuilder builder;
+  g_variant_builder_init (&builder, G_VARIANT_TYPE ("a(ayay)"));
+
+  GVariant* res = NULL;
+  try {
+    res = filter_xattrs_impl(repo, relpath, file_info, user_data);
```

Comment:
This could just be `return filter_xattrs_impl(...)` right?

Reply:
Oh right, absolutely. My brain is still not used to reason with exceptions.